agent/lib_monolog.c

```diff
@@ -0,0 +1,387 @@
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "php_agent.h"
+#include "php_call.h"
+#include "php_user_instrument.h"
+#include "php_datastore.h"
+#include "php_execute.h"
+#include "php_hash.h"
+#include "php_wrapper.h"
+#include "fw_hooks.h"
+#include "fw_support.h"
+#include "nr_datastore_instance.h"
+#include "nr_segment_datastore.h"
+#include "util_logging.h"
+#include "util_memory.h"
+#include "util_strings.h"
+#include "util_sleep.h"
+
+/*
+ * Purpose : Return a copy of Monolog\Logger::API.
```

Comment:
The purpose seems different than what the return indicates.

Reply:
It is a leftover from initial version of this function. I'll change the description.

Reply:
@zsistla This has been addressed in 7cbdff18a4ac99dc5e165d7c1f000140f1b851b6